File: rest-api/src/main/java/com/kt/kol/api/comment/model/CommentDTO.java
package com.kt.kol.api.comment.model;

import java.time.LocalDateTime;

public record CommentDTO(
    Long id,
    String content,
    Long postId,
    Long userId,
    String username,
    Long parentId,
    LocalDateTime createdAt,
    LocalDateTime updatedAt
) {}

File: rest-api/src/main/kotlin/com/kt/kol/api/comment/model/CommentDTO.kt
package com.kt.kol.api.comment.model

import java.time.LocalDateTime

data class CommentDTO(
    val id: Long? = null,
    val content: String? = null,
    val postId: Long? = null,
    val userId: Long? = null,
    val username: String? = null,
    val parentId: Long? = null,
    val createdAt: LocalDateTime? = null,
    val updatedAt: LocalDateTime? = null
)